reaper/client.go

```diff
@@ -217,6 +222,36 @@ func (c *Client) DeleteCluster(ctx context.Context, cluster string) error {
 	return nil
 }
 
+func (c *Client) RepairSchedules(ctx context.Context) ([]RepairSchedule, error) {
+	rel := &url.URL{Path: "/repair_schedule"}
+	return c.fetchRepairSchedules(ctx, rel)
+}
+
+func (c *Client) RepairSchedulesPerCluster(ctx context.Context, clusterName string) ([]RepairSchedule, error) {
```

Comment:
Maybe consider renaming this to `RepairSchedulesForCluster`. The `Per` in `RepairSchedulesPerCluster` suggests multiple clusters.